package.json

```diff
@@ -1,38 +1,57 @@
 {
   "name": "setup-tflint-action",
-  "version": "0.1.0",
+  "version": "1.0.4",
   "description": "Install and setup TFLint executable Action",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint .",
-    "prepare": "ncc build index.js -o dist --no-source-map-register",
-    "test": "jest",
-    "all": "npm run lint && npm run prepare && npm run test"
+    "lint": "eslint --fix . src test",
+    "build": "ncc build index.js -o dist --no-source-map-register",
+    "test": "jest test",
+    "prepublishOnly": "pinst --disable",
+    "postinstall": "husky install",
+    "postpublish": "pinst --enable"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix"
+    ],
+    "*.ts": [
+      "eslint --fix"
+    ]
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/lablabs/setup-tflint-action.git"
+    "url": "git+https://github.com/jgeurts/setup-tflint-action.git"
   },
   "keywords": [
     "GitHub",
     "Actions",
     "TFLint",
     "Terraform"
   ],
-  "author": "Labyrinth Labs",
+  "author": "jgeurts",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/lablabs/setup-tflint-action/issues"
-  },
-  "homepage": "https://github.com/lablabs/setup-tflint-action#readme",
+  "homepage": "https://github.com/jgeurts/setup-tflint-action#readme",
```

Comment:
Should stay at git+https://github.com/lablabs/setup-tflint-action.git if we're going to merge this into lablabs:master